runtime/stream.h

```diff
@@ -216,6 +217,6 @@ strmGetCurrFileNum(strm_t *pStrm) {
 /* prototypes */
 PROTOTYPEObjClassInit(strm);
 rsRetVal strmMultiFileSeek(strm_t *pThis, int fileNum, off64_t offs, off64_t *bytesDel);
-rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF);
+rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF, off64_t *offset);
```

Comment:
This is a binary incompatible change to the plugin API, but it looks like imfile is the only one that uses this API, at least in the distributed rsyslog source code.  What is the policy here?  Usually this entails adding a new function e.g. strmReadLineWithOffset or something like that, and change the implementation of strmReadLine to call strmReadLineWithOffset with NULL or a dummy parameter for the offset argument.